Modules/_io/_iomodule.c

```diff
@@ -368,6 +369,8 @@ _io_open_impl(PyObject *module, PyObject *file, const char *mode,
         if (blksize_obj == NULL)
             goto error;
         buffering = PyLong_AsLong(blksize_obj);
+        if (buffering < DEFAULT_BUFFER_SIZE)
```

Comment:
All new `if` / code structure needs to do curly braces (the _io_open_impl code is fairly old, hence doesn't have a lot)

see: https://peps.python.org/pep-0007/